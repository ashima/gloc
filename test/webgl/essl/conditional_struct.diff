--- conditional_struct.true	2011-11-14 15:58:53.000000000 -0800
+++ conditional_struct.out	2011-11-17 16:38:56.000000000 -0800
@@ -1 +1,8 @@
-error
+precision highp float;
+
+void main() {
+
+  if (false) struct s { int k; } q = s(0);
+
+  gl_FragColor = vec4(float(s(q.k).k));
+}
