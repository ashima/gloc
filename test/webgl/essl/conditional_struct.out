precision highp float;

void main() {

  if (false) struct s { int k; } q = s(0);

  gl_FragColor = vec4(float(s(q.k).k));
}
